### DISCUSSION TITLE
"All symbols" is hard to use in mobile webpage

### DISCUSSION THREAD

**Discussion by oxygen-dioxide** [community/community#45280] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In github mobile page's "All symbols" menu, when I exit the menu, the page automatically rolls back to where I viewed before the search. I have to manually remember the line number and jump there.

**Comment by oxygen-dioxide** [comment]
Here is a screenshot video: https://1drv.ms/v/s!AgZK3YWSBqjhnivPVeNMXrLIZPDd?e=L6JDcP

**Reply by milemons** [reply]
Thanks for the report! We will work on a fix!

**Comment by DavidJCobb** [comment]
Since GitHub has left this bug out to rot for the last two years, I've hacked together [a quick-and-dirty userscript](https://gist.github.com/DavidJCobb/ac17988b02cab5d7b028fd133647f432) to fix it. There's an explanation of the bug's most likely cause over there too, if anyone at GitHub would like to actually work on a fix now.
